### PR TITLE
Add Favicon

### LINKED ISSUE #20: Add Favicon

### PR DESCRIPTION
**Pull request by hf-krechan**
I just followed this blog entry: https://evilmartians.com/chronicles/how-to-favicon-in-2021-six-files-that-fit-most-needs

Not totally sure if this is the "angular" way ^^

Fixes #20

### CODE REVIEW

File: src/app/shared/components/header/header.component.html [hunk 1]
```diff
@@ -1,16 +1,13 @@
 <header class="bg-primary">
-  <nav
-    class="mx-auto flex max-w-7xl items-center justify-between p-6 lg:px-8"
-    aria-label="Global"
-  >
+  <link rel="icon" href="/assets/favicon/favicon.ico" sizes="32x32">
+  <link rel="icon" href="/assets/favicon/hf_logo.svg" type="image/svg+xml">
+  <link rel="apple-touch-icon" href="/assets/favicon/apple-touch-icon.png"><!-- 180×180 -->
+  <link rel="manifest" href="/assets/favicon/manifest.webmanifest">
+  <nav class="mx-auto flex max-w-7xl items-center justify-between p-6 lg:px-8" aria-label="Global">
```
Comment (TommyBom) [inline src/app/shared/components/header/header.component.html]: header != head

Please use the `index.html` file. This component renders a header element in the html body.

**Review by TommyBom — CHANGES_REQUESTED** [review]
please use index.html head file for link tags

File: src/site.webmanifest [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    {
+      "src": "/assets/favicon/hf_logo_512x512.png",
+      "sizes": "512x512",
+      "type": "image/png"
+    }
+  ],
+  "theme_color": "#EBBEC1",
+  "background_color": "#EBBEC1",
+  "display": "standalone"
+}
```
Comment (TommyBom) [inline src/site.webmanifest:19]: We should leave this out. If you want pwa functionality we should follow this guide: https://angular.io/guide/service-worker-getting-started#adding-a-service-worker-to-your-project

**Review by TommyBom — CHANGES_REQUESTED** [review]

File: src/site.webmanifest (hunk 2 above)
Comment (hf-krechan) [reply src/site.webmanifest]: removed in c5567d1f663ccfb759bfc35a77e6a573e7bb8a5f

File: src/app/shared/components/header/header.component.html (hunk 1 above)
Comment (hf-krechan) [reply src/app/shared/components/header/header.component.html]: moved in 566d0ad454b8184a3b2c4259937eda2c9444f536

Comment (TommyBom) [reply src/app/shared/components/header/header.component.html]: nope still there

Comment (TommyBom) [reply src/app/shared/components/header/header.component.html]: you copied it to the index.html, but forgot to remove it here

Comment (hf-krechan) [reply src/app/shared/components/header/header.component.html:2]: I removed it, but I got some issues with git.
But now it should be gone ^^

d59dd546d87a91c85b706de9e834f21e20eb0cca

**Review by TommyBom — APPROVED** [review]

**Pull request merged**